emissary.yaml

```diff
@@ -0,0 +1,100 @@
+package:
+  name: emissary
+  version: 3.9.1
+  epoch: 0
+  description: "open source Kubernetes-native API gateway for microservices built on the Envoy Proxy"
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - datawire-envoy-privileged
+      - git
+      - python3
+
+environment:
+  contents:
+    packages:
+      - go
+      - iproute2
+      - libcap-utils
+      - py3-pip
+      - python3
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/emissary-ingress/emissary
+      tag: v${{package.version}}
+      expected-commit: 6e2ca35c11d124da26df8dd029c33954960b15ae
+
+  # Go binaries
+  - uses: go/build
+    with:
+      packages: ./cmd/apiext
+      output: apiext
+
+  - uses: go/build
+    with:
+      packages: ./cmd/busyambassador
+      output: busyambassador
+
+  - uses: go/build
+    with:
+      packages: ./cmd/capabilities_wrapper
+      output: wrapper
+
+  - name: wrapper-setcap
+    # See https://github.com/emissary-ingress/emissary/blob/ac2dc64c6621cd8ec5617f3328544364bdd3fb01/build-aux/Dockerfile#L129-L137
+    runs: |
+      setcap cap_net_bind_service=p ${{targets.contextdir}}/usr/bin/wrapper
+
+  - uses: go/build
+    with:
+      packages: ./cmd/kubestatus
+      output: kubestatus
+
+  - name: Python Build
+    runs: |
+      make python/ambassador.version
+
+      # Install Python dependencies
+      python3 -m venv venv
```

Comment:
We should install deps without a venv now

Reply:
We should update https://eng.inky.wtf/docs/images/package-and-image-guides/building-package-walkthrough-python/ - that's what made me think we needed venv.

What's the recommended way to do this?

Reply:
Let's definitely update the docs. It's the same process. You just wouldn't create the virtual - environment; you'd use the hosts pip directly

Reply:
Got things working. PTAL!